doc/modules/ROOT/pages/getting-started.adoc

```diff
@@ -91,6 +91,28 @@ Or we could change the database we are targeting later:
 gds.set_database("my-db")
 ----
 
+=== Configure Arrow parameters
+
+If Arrow is available on the server, we can provide the `GraphDataScience` constructor with several keyword parameters to configure the connection:
+
+* arrow_disable_server_verification - A flag that indicates that, if the flight client is connecting with
+        TLS, that it skips server verification. If this is enabled, all
```

Comment:
I would rephrase this a bit. What does "other TLS settings are overridden" mean? Also, when would one want to use this? When a self-signed certificate is used?

Reply:
We are wrapping options that you can configure on the pyArrow flight client (https://arrow.apache.org/docs/python/generated/pyarrow.flight.FlightClient.html#pyarrow.flight.FlightClient).

I will try to find out when it makes sense :)

Reply:
Reading more i think its mainly ease of use in the arrow setup